src/components/manage/Blocks/Image/View.jsx

```diff
@@ -30,28 +31,33 @@ const View = ({ data, detached }) => (
       <>
         {(() => {
           const image = (
-            <img
-              className={cx({ 'full-width': data.align === 'full' })}
-              src={
-                isInternalURL(data.url)
-                  ? // Backwards compat in the case that the block is storing the full server URL
-                    (() => {
-                      if (data.size === 'l')
-                        return `${flattenToAppURL(data.url)}/@@images/image`;
-                      if (data.size === 'm')
-                        return `${flattenToAppURL(
-                          data.url,
-                        )}/@@images/image/preview`;
-                      if (data.size === 's')
-                        return `${flattenToAppURL(
-                          data.url,
-                        )}/@@images/image/mini`;
-                      return `${flattenToAppURL(data.url)}/@@images/image`;
-                    })()
-                  : data.url
-              }
-              alt={data.alt || ''}
-            />
+            <picture>
+              <source
+                srcSet={`
+            ${flattenToAppURL(data.url)}/@@images/image/mini 200w,
```

Comment:
Maybe save ``flattenToAppURL(data.url)`` to an intermediary var? This way code is shorter and optimized